#commands
## slave_id is from 1
## servo_index is from 0
uint8 SET_SLAVE_ID = 0
# [slave_id, new_slave_id]
uint8 SET_IMU_SEND_FLAG = 1
# [slave_id, flag]
uint8 SET_SERVO_HOMING_OFFSET = 2
# [slave_id, servo_index, joint_offset] !torque disable!
uint8 SET_SERVO_PID_GAIN = 3
# [slave_id, servo_index, p_gain, i_gain, d_gain]
uint8 SET_SERVO_PROFILE_VEL = 4
# [slave_id, servo_index, profile_vel]
uint8 SET_SERVO_SEND_DATA_FLAG = 5
# [slave_id, servo_index, flag]
uint8 SET_SERVO_CURRENT_LIMIT = 6
# [slave_id, servo_index, current_limit] !torque disable!
uint8 REBOOT = 7
# [slave_id]
uint8 SET_DYNAMIXEL_TTL_RS485_MIXED = 8
# [slave_id, flag]
uint8 SET_SERVO_EXTERNAL_ENCODER_FLAG = 9
# [slave_id, servo_index, flag]
uint8 SET_SERVO_RESOLUTION_RATIO = 10
# [slave_id, servo_index, joint_resolution, servo_resolution]

uint8 SET_SERVO_PULLEY_SKIP_THRESH = 12
# [slave_id, thresh]

uint8 SET_SERVO_INTERNAL_OFFSET_LPF_RATE = 13
# [slave_id, rate percentage (rate * 100)]


uint8 command
int32[] data
---
bool success
